relayer/src/chain/cosmos/batch.rs

```diff
@@ -187,11 +219,19 @@ fn batch_messages(
 #[cfg(test)]
 mod tests {
     use super::batch_messages;
-    use crate::config::types::{MaxMsgNum, MaxTxSize};
+    use crate::chain::cosmos::types::account::Account;
+    use crate::chain::cosmos::types::config::TxConfig;
+    use crate::config::types::{MaxMsgNum, MaxTxSize, Memo};
+    use crate::keyring::KeyEntry;
     use ibc_proto::google::protobuf::Any;
 
+    fn test_fixture() -> (TxConfig, KeyEntry, Account) {
+        todo!()
+    }
```

Comment:
@soareschen Is there anything in the test framework to provide usable values here? Unfortunately, the unit tests become more complicated.

Reply:
There could be a mock instance for `TxConfig`, if you don't call the API URLs. For `KeyEntry` you would have to construct a private key pair. For `Account` the main essential information is the account sequence, which could be set to something like `0` or `999999`.

Reply:
Good, I'm creating mock values (not pushed yet), but had doubts about all fields of `TxConfig`.